### PR TITLE
corrected typos

### PR DESCRIPTION
**Pull request by Pascakk**

### CODE REVIEW

File: _posts/2020-06-27-log-day6.md [hunk 1]
```diff
@@ -18,16 +18,16 @@ tags:
...
 
 * Jean-Marie and Thomas
     - step by step implementation of collisions or Isaac : first with one rock, then several rocks, now several elemets blocking or not
     - added collisions for walls in order to prevent Isaac from leaving the screen
     - set up the mecanism letting Isaac look in the right direction
-    - refactoring collisions for better performance and in order to allowe Isaac to "slide" against a blocking element
+    - refactoring collisions for better performance and in order to allow Isaac to "slide" against a blocking element
 
 * Mélissa and Dorian
-    - custom version of the score "Sacrificial" from the original game, adapted to a 2 channel player
```
Comment (DorianXGH) [inline _posts/2020-06-27-log-day6.md:31]: score is the better term, the score OF

Comment (Pascakk) [reply _posts/2020-06-27-log-day6.md:31]: okay, I will fix that

Comment (DorianXGH) [reply _posts/2020-06-27-log-day6.md:31]: the score OOOOOOOOFFFFFFF

Comment (DorianXGH) [reply _posts/2020-06-27-log-day6.md:31]: Are you kidding me ?

**Comment by Pascakk** [comment]
I think I actually am

**Comment by Pascakk** [comment]
hehe are we there yet ?

**Comment by Pascakk** [comment]
Time to celebrate

**Pull request merged**